client/src/components/MessageList/Style.module.css

```diff
@@ -0,0 +1,20 @@
+.messageContainer {
+  background-color: #f2f2f2;
+  padding: 10px;
+  font-family: inherit;
```

Comment:
`font-family: inherit;` not required. By default, it'll inherit.

Reply:
How is that?

Reply:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance#Inheritance